pages/common/aria2.md

```diff
@@ -1,6 +1,6 @@
 # aria2
 
-> A lightweight multi-protocol & multi-source command-line download utility.
+> A lightweight multi-protocol & multi-source command line download utility.
```

Comment:
I thought `command-line` was OK. The official git documentation uses both `command line` and `command-line` ([example](https://git-scm.com/docs/git)). Any references? :innocent:

Reply:
Personally, I prefer "command-line" (not sure why 🤔).  But it's a tie! 🤞🏻

These stats are taken from the latest `main` @ 5ba22052b74d11926b66444e53014e5c74630fe1:
```
command line: 116
command-line: 113
commandline:  65
```

Reply:
`commandline` is wrong
I also vote for `command-line`, though if you googe around, in textual context its more referred to as `command line`. But I like `command-line` better.

Reply:
It really depends on context as to which one I prefer. In this particular instance, I prefer it with a dash - but that is not transferable to other cases, which I would need to individually review.